wii-u-gc-adapter.c

```diff
@@ -605,10 +626,15 @@ int main(int argc, char *argv[])
    if (count > 0)
       libusb_free_device_list(devices, 1);
 
+   if (libusb_has_capability(LIBUSB_CAP_HAS_HOTPLUG) == 0)
+      fprintf(stderr, "missing hotplugging capability.\n");
```

Comment:
In case hotplug capability isn't available, we should probably not even try to register the callback since it's gonna fail anyway.